Backend/Lyzer/Clients/JolpicaClient.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Text.Json;
+using RestSharp;
+using Newtonsoft.Json;
+using Lyzer.Common.Constants;
+using Lyzer.Common.DTO;
+using System.Runtime.Serialization;
+
+namespace Lyzer.Clients
+{
+    public class JolpicaClient
+    {
+        private readonly ILogger<JolpicaClient> _logger;
+        private readonly RestClient _client;
+
+        public JolpicaClient(ILogger<JolpicaClient> logger)
+        {
+            RestClientOptions options = new RestClientOptions(URIConstants.Jolpica.BaseUri);
+            _client = new RestClient(options);
+            _logger = logger;
+        }
+
+        public async Task<DriverStandingsDTO> GetCurrentDriverStandings()
+        {
+            string requestPath = String.Format(URIConstants.Jolpica.DriverStandingsUri, "current");
```

Comment:
Wondering if having a folder called URIConstants and then having a Jolipca.cs file inside there, if it would look cleaner? What are your thoughts?

Reply:
I like the idea of that, I will get to it :)

Reply:
Well actually hang on, will we be using other APIs in the future that we are certain of.
Just thinking, its a bit premature if we are just going to end up having one file in the folder?

Maybe not tho, idk?

Reply:
@Pranav-Zodes-44